tests/ts_cluster_create_table_via_sql_SUITE.erl

```diff
@@ -0,0 +1,140 @@
+%% -------------------------------------------------------------------
+%%
+%% Copyright (c) 2016 Basho Technologies, Inc.
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+%% -------------------------------------------------------------------
+
+-module(ts_cluster_create_table_via_sql_SUITE).
+-compile(export_all).
+
+-include_lib("common_test/include/ct.hrl").
+-include_lib("eunit/include/eunit.hrl").
+
+suite() ->
+    [{timetrap,{minutes, 10}}].
+
+-define(CUSTOM_NVAL, 4).
+-define(CUSTOM_MVAL, <<"bo o''oo">>).
+
+init_per_suite(Config) ->
+    Cluster = ts_util:build_cluster(multiple),
+    [{cluster, Cluster} | Config].
+
+end_per_suite(_Config) ->
+    ok.
+
+all() ->
+    [create_test,
+     re_create_fail_test,
+     describe_test,
+     get_put_data_test,
+     create_fail_because_bad_properties_test,
+     get_set_property_test].
+
+
+create_fail_because_bad_properties_test(Ctx) ->
+    C = client_pid(Ctx),
+    BadDDL =
+        io_lib:format(
+          "~s WITH (m_val = ~s)",
+          [ddl_common(), "plain_id"]),
+    Got = riakc_ts:query(C, BadDDL),
+    ?assertMatch({error, {1020, _}}, Got),
+    io:format("Not created table with bad properties: ~p\n", [Got]),
+    pass.
+
+create_test(Ctx) ->
+    C = client_pid(Ctx),
+    GoodDDL =
+        io_lib:format(
+          "~s WITH (n_val=~b, m_val = '~s')",
+          [ddl_common(), ?CUSTOM_NVAL, ?CUSTOM_MVAL]),
+    Got1 = riakc_ts:query(C, GoodDDL),
+    ?assertEqual({[],[]}, Got1),
+    io:format("Created table via query: ~p\n", [Got1]),
+    pass.
+
+re_create_fail_test(Ctx) ->
+    C = client_pid(Ctx),
+    Got = riakc_ts:query(C, ddl_common()),
+    ?assertMatch({error, {1014, _}}, Got),
+    io:format("Not created same table via query: ~p\n", [Got]),
```

Comment:
This logging is confusing while the test is running. It is not clear whether this is logging a good thing that happened or a bad thing without looking at the code. We shouldn't log anything extra when the test passes.